galata/test/jupyterlab/kernel.test.ts

```diff
@@ -104,6 +104,72 @@ test.describe('Kernel', () => {
         'Python 3 (ipykernel)'
       );
     });
+
+    test('Should support opening subshell in separate code console', async ({
+      page
+    }) => {
+      // Open new notebook
+      await page.menu.clickMenuItem('File>New>Notebook');
+      await page
+        .getByRole('tab', { name: 'Untitled.ipynb' })
+        .waitFor({ state: 'detached' });
+      await page.getByRole('button', { name: 'Select Kernel' }).click();
+
+      // Run %subshell in notebook
+      const notebook = page.locator('.jp-Notebook');
+      await notebook.waitFor();
+      await page.notebook.setCell(0, 'code', '%subshell');
+      await page.notebook.runCell(0);
+
+      const output1 = notebook.locator('.jp-OutputArea-output').locator('pre');
+      const text1 = (await output1.innerText()).split('\n');
+
+      // Confirm that subshells are supported by %subshell printing something useful.
+      // Subshell ID for this main shell is None, and no subshells have been created
+      // yet so the subshell list is empty.
+      expect(text1[0]).toEqual('subshell id: None');
+      expect(text1[5]).toEqual('subshell list: []');
+
+      // Open subshell console
+      await output1.click({ button: 'right' });
+      await page
+        .getByRole('menuitem', { name: 'New Subshell Console for Notebook' })
+        .click();
+
+      // Run %subshell in console
+      const subshellConsole = page.locator('.jp-CodeConsole');
+      await subshellConsole.waitFor();
+      await page.notebook.setCell(0, 'code', '%subshell');
+      await page.notebook.runCell(0);
```

Comment:
I think these cannot be run using the `notebook` helper, and this is why the tests started failing. Notice how my suggestion was only on lines 121-122 and not on these lines.